examples/custom_transform_and_model.ipynb

```diff
@@ -202,6 +202,16 @@
     "ts.head(5)"
```

Comment:
<blockquote>The one thing we need to implement in both cases is only the method <code>get_regressors_info</code> — it should return the created regressors, created by the Transform. </blockquote><p>This phrase looks strange. May be we should rewrite it somehow. Created was used two times.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>